internal/ethapi/transaction_args.go

```diff
@@ -117,7 +117,7 @@ func (args *TransactionArgs) setDefaults(ctx context.Context, b Backend) error {
 				if err != nil {
 					return err
 				}
-				if b.ChainConfig().IsLondon(head.Number) {
+				if b.ChainConfig().IsBanach(head.Number) {
```

Comment:
in the above line (which cannot be commented on), do we need to change the error logs from `london` to `banach`?

this log:
```
maxFeePerGas or maxPriorityFeePerGas specified but london is not active yet
```

Reply:
Fixed in [80fd1ce](https://github.com/scroll-tech/go-ethereum/pull/634/commits/80fd1ce2687af1408d26cd36bcf418959b05b0df).